docs/data/toolpad/core/components/sign-in-page/sign-in-page.md

```diff
@@ -18,29 +18,7 @@ The `SignInPage` component can be set up with an OAuth provider by passing in a
 
 :::info
 
-The following providers are supported and maintained by default:
-
-- Google
-- GitHub
-- Facebook
-- Microsoft (Entra ID)
-- Apple
-- Auth0
-- AWS Cognito
-- GitLab
-- Instagram
-- LINE
-- Okta
-- FusionAuth
-- Twitter
-- TikTok
-- LinkedIn
-- Slack
-- Spotify
-- Twitch
-- Discord
-- Keycloak
-- Credentials (username/password)
+Find the list of supported providers in the [API reference](/toolpad/core/api/sign-in-page/#sign-in-page-prop-providers).
```

Comment:
This feels worse. Can't we do the opposite? Keep the list here and just call it a string in the API docs?